cmd/evm/internal/t8ntool/execution.go

```diff
@@ -36,12 +39,14 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/ethereum/go-ethereum/trie"
+	"github.com/ethereum/go-verkle"
 	"golang.org/x/crypto/sha3"
 )
 
 type Prestate struct {
-	Env stEnv             `json:"env"`
-	Pre core.GenesisAlloc `json:"pre"`
+	Env stEnv                         `json:"env"`
+	Pre core.GenesisAlloc             `json:"pre"`
+	VKT map[common.Hash]hexutil.Bytes `json:"vkt,omitempty"`
```

Comment:
Worth renaming this to `PreMPT` and `PreVKT`?

Reply:
that's what I did, the testing team reverted it so there must be reason not to do that.